src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(value) {
```

Comment:
Consider adding a check to ensure that the `value` is a number before adding it to `sum`. This will prevent unexpected behavior if non-numeric values are passed.